src/v2i-hub/TelematicBridgePlugin/REAADME.md

```diff
@@ -0,0 +1,94 @@
+## NATS Publisher/Subscriber
+### NATS Connections and registration
```

Comment:
REAME file is misspelled

Reply:
Addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/567/commits/71ac033b94eec7ea112cb5ead2d680a29b769772